src/views/RecIM/components/List/index.jsx

```diff
@@ -42,6 +42,28 @@ const ParticipantList = ({
   showInactive,
   callbackFunction,
 }) => {
+  const [newCaptain, setNewCaptain] = useState();
+  const [teamId, setTeamID] = useState();
+
+  useEffect(() => {
+    // find the old capitain to demote
+    participants.map(async (participant) => {
+      if (
+        participants.length > 0 &&
+        participant.Role === 'Team-captain/Creator' &&
+        participant !== newCaptain &&
+        teamId !== undefined
+      ) {
+        let editedParticipant = {
+          Username: participant.Username,
+          RoleTypeID: 3,
+        }; // Role 3 is member
+        await editTeamParticipant(teamId, editedParticipant);
+      }
+      callbackFunction((bool) => !bool);
```

Comment:
I wouldn't name a variable after a common type, `boolean`. That will confuse other developers. It would be better to do something simple like a character.
```suggestion
      callbackFunction((b) => !b);
```